packages/react-reconciler/src/ReactFiberCompleteWork.js

```diff
@@ -618,15 +638,18 @@ function updateHostText(
       // If the text content differs, we'll create a new text instance for it.
       const rootContainerInstance = getRootHostContainer();
       const currentHostContext = getHostContext();
+      markCloned(workInProgress);
       workInProgress.stateNode = createTextInstance(
         newText,
         rootContainerInstance,
         currentHostContext,
         workInProgress,
       );
-      // We'll have to mark it as having an effect, even though we won't use the effect for anything.
-      // This lets the parents know that at least one of their children has changed.
-      markUpdate(workInProgress);
+      if (!enablePersistedModeClonedFlag) {
+        // We'll have to mark it as having an effect, even though we won't use the effect for anything.
+        // This lets the parents know that at least one of their children has changed.
+        markUpdate(workInProgress);
+      }
```

Comment:
This may be right but it's a little confusing that markCloned and markUpdate are not apparently mutually exclusive (when they actually are given the implementation of markCloned)

Reply:
This is actually the core of the PR. We can have an `Update` for example due to effect dependencies changing, but do not want to trigger a clone up the tree indicated by the `Cloned` flag.

Previously they were both grouped together which means we'd do a full subtree clone if there was an effect dependency change in the subtree.

Even with the feature flag an effect change will still set the `Update` flag, but `Update` (behind flag) no longer causes a clone.